modules/mobile/app/workers/mobile/v0/vaccines_updater_job.rb

```diff
@@ -0,0 +1,99 @@
+# frozen_string_literal: true
+
+module Mobile
+  module V0
+    # This job is run daily and pulls data from the CDC to create vaccine records
+    class VaccinesUpdaterJob
+      include Sidekiq::Worker
+
```

Comment:
The default number of sidekiq retry attempts is 25 (which with its [exponential back-off algorithm](https://github.com/mperham/sidekiq/wiki/Error-Handling#automatic-job-retry) ends up being 21 days) since this is a daily job I think 3-10 would keep it within one day, but we probably only need the lower end.
```ruby
sidekiq_options retry: 3
```

Reply:
That's a good point. Added.